nixos/modules/programs/ssh.nix

```diff
@@ -310,6 +310,8 @@ in
           XAuthLocation ${pkgs.xorg.xauth}/bin/xauth
         ''}
 
+        EnableEscapeCommandline yes
+
```

Comment:
This change needs to be in its own commit, and ideally it would be exposed as an option.

Though delaying the second matter is fine by me.

Reply:
Don't so sure if these should be enabled per default... In previous versions it was enabled by default since these not anymore as far as I understood. It could be get disabled per default because of the vulnerability in pre 9.2 versions.
The description:

ssh(1): add a new EnableEscapeCommandline ssh_config(5) option that controls whether the client-side ~C escape sequence that provides a command-line is available. Among other things, the ~C command-line could be used to add additional port-forwards at runtime. This option defaults to "no", disabling the ~C command-line that was previously enabled by default. Turning off the command-line allows platforms that support sandboxing of the ssh(1) client (currently only OpenBSD) to use a stricter default sandbox policy

Reply:
> This change needs to be in its own commit, and ideally it would be exposed as an option.
> 
> Though delaying the second matter is fine by me.

If at all.

What is the reasoning behind reverting this upstream change?

Why is there no explanation whatsoever?